pastas/version.py

```diff
@@ -44,7 +47,7 @@ def show_versions(lmfit: bool = True, latexify: bool = True) -> None:
         f"NumPy version: {metadata.version('numpy')}\n"
         f"Pandas version: {metadata.version('pandas')}\n"
         f"SciPy version: {metadata.version('scipy')}\n"
-        f"Matplotlib version: {metadata.version('matplotlib')}\n"
+        f"Matplotlib version: {mplversion}\n"
```

Comment:
maybe:     
mplversion= metadata.version("matplotlib") if metadata.version("matplotlib") is not None else mplversion